app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Recipe #<%= @recipe.id %></h1>
+
+<h3>Name: <%= @recipe.name %></h3>
+<h3>Complexity: <%= @recipe.complexity %></h3>
+<h3>Genre: <%= @recipe.genre %></h3>
+
+<h2>Ingredients for Enchiladas</h2>
+<h3><% @recipe.ingredients.each do |ingredient| %></h3>
```

Comment:
Non-rendered erb doesn't have to have HTML. So you can remove the `<h3>` from line 8.